.github/workflows/build-mac.yml

```diff
@@ -80,13 +80,8 @@ jobs:
           rm -rf tmp
       - name: Capture version before localization
         run: echo "SETUPTOOLS_SCM_PRETEND_VERSION=$(python -W ignore distro.py --version)" >> $GITHUB_ENV
```

Comment:
You should be able to remove the `SETUPTOOLS_SCM_PRETEND_VERSION` env var from the mac and windows workflows now that's handled in the build scripts.

Reply:
Right, didn't mean to push. Was trying to get git history clean to make sure the version string is clean locally.